### DISCUSSION TITLE
I dkn why sometimes I cannot jump to the dependencies by ctrl +click, and it keeps loading all the time

### DISCUSSION THREAD

**Discussion by CoheeYang** [Cyfrin/foundry-full-course-cu#3118] [General]
![image](https://github.com/user-attachments/assets/e2d373ef-7914-4451-bc6e-a274c27ebfc2)
I dkn why sometimes I cannot jump to the dependencies by ctrl +click, and it keeps loading all the time

**Comment by EngrPips** [comment]
That might be `vscode` acting up. Maybe close and reopen `vscode`
